app/src/main/java/study_dev/testbed/studyhopper/models/Profile.java

```diff
@@ -7,32 +7,45 @@
 public class Profile {
     String firstName;
     String lastName;
-    Date dob;
-    String gender;
-    String university;
-    Timestamp lastLogin;
-
-    Profile() {
 
-    }
+    String email;
 
-    public Profile(String firstName, String lastName, Date dob, String gender, String university) {
+    public Profile(String firstName, String lastName, Date dob, String gender, String university, String email) {
```

Comment:
One suggestion will be adding a screen name to profiles that the user can change. However it is something more for a future update.

Reply:
that can be done directly from Authentication. It has a display name property. But yeah I can add it when we do profile pics